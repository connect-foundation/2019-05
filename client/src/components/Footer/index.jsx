import React from 'react';
import './Footer.scss';

const Footer = () => {
  return (
    <div className="footer">
      <footer className="footer__message">
        I'm your footer
      </footer>
    </div>
  );
};

export default Footer;