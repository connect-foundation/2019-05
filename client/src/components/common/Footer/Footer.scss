.footer {
  background-color: #17252A;
  color: whitesmoke;
  display: flex;
  margin: 0 auto;
  justify-content: center;

  &__list-items {
    list-style-type: none;
  }
}