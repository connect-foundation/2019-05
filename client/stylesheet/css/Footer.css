.footer {
  background-color: transparent;
  position: absolute;
  bottom: 0px;
  color: whitesmoke;
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  margin: 0 auto;
  -webkit-box-pack: center;
      -ms-flex-pack: center;
          justify-content: center;
  -webkit-box-align: center;
      -ms-flex-align: center;
          align-items: center;
  height: 80px;
  width: 100%;
}

.footer__list-items {
  list-style-type: none;
}
