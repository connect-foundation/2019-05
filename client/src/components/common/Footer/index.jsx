import React, { Fragment } from 'react';
import './Footer.scss';


const Footer = () => (
  <footer className="footer">
    <FooterCategoryBlock title="Footer" contents={["this", "is", "footer"]} />
  </footer>
);


const FooterCategoryBlock = ({ title, contents }) => (
  <Fragment className="footer__list">
    <h1 className="footer__list-header">{title}</h1>
    <ul className="footer__list-items">
      {contents.map(content => <li>{content}</li>)}
    </ul>
  </Fragment>
)


export default Footer;