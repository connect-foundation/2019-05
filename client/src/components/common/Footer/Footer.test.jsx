import React from 'react';
import { render } from '@testing-library/react';
import Footer from '.';

describe('Footer component', () => {
  it('matches with the snapshot', () => {
    const { getByText } = render(<Footer />);
    const footer = getByText(/^Copy/);
    expect(footer).toMatchSnapshot();
  });
});
