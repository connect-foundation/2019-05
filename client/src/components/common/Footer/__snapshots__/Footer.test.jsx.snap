// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matches with the snapshot 1`] = `
<footer
  class="footer"
>
  Copyright 2019. team Underdoggs. All rights reserved.
</footer>
`;
